vllm/v1/engine/output_processor.py

```diff
@@ -378,8 +382,24 @@ def abort_requests(
         self,
         request_ids: Iterable[str],
```

Comment:
We could also consider adding an `internal: bool = False` arg here to allow caller to specify whether or not the ids are internal or external. This might be a bit more robust, for all the places we call abort internally, we should be able to use the internal id and set it to `True`, and external usage shouldn't be affected.